tasks/jest/matchers/toHaveNoACViolations.js

```diff
@@ -0,0 +1,19 @@
+import aChecker from 'accessibility-checker';
+
+('use strict');
```

Comment:
```suggestion
'use strict';
```

This most likely got wrapped since there is an `import` at the top-level. Updating that `import` to `require` and placing it after `'use strict'` should do the trick 👍